server/main.go

```diff
@@ -200,6 +201,7 @@ func main() {
 		})
 
 		r.Route("/", func(r chi.Router) {
+			r.Head("/", pingDB)
```

Comment:
Can we just return static 200 from this one? That way the status page can have separate statuses for web and api.